src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+/**
+ * Represents a User interface
+ */
+public class Ui {
+    private Scanner scanner;
+
+    /**
+     * Constructs a `Ui` object and initializes the scanner for user input.
+     */
+    public Ui() {
+        scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Displays a welcome message when the program starts.
+     */
+    public void showWelcomeMessage() {
+        System.out.println("Hi! This is your AI assistant LoyBoy!");
```

Comment:
You can store these strings as a final variable as you did in Storage. This is still okay though.